runtime/src/accounts_db.rs

```diff
@@ -1518,6 +1518,14 @@ struct IndexAccountMapEntry<'a> {
 
 type GenerateIndexAccountsMap<'a> = HashMap<Pubkey, IndexAccountMapEntry<'a>>;
 
+#[derive(Debug, Default, Clone, Copy)]
+struct GenerateIndexForSlotResult {
```

Comment:
`GeneratedIndexForSlotResult`?

Reply:
Yes, is for getting the accounts data len during deserialization of a snapshot.